experimenter/experimenter/nimbus_ui_new/forms.py

```diff
@@ -600,14 +611,22 @@ def __init__(self, *args, **kwargs):
         self.fields["feature_configs"].queryset = NimbusFeatureConfig.objects.filter(
             application=self.instance.application
         ).order_by("slug")
+        show_errors = ""
+        if (
+            hasattr(self, "request")
+            and self.request
+            and self.request.GET.get("show_errors") == "true"
+        ):
+            show_errors = "?show_errors=true"
 
         update_on_change_attrs = {
             "hx-post": reverse(
                 "nimbus-new-partial-update-branches", kwargs={"slug": self.instance.slug}
-            ),
+            )
+            + show_errors,
```

Comment:
Oh we really should never use `+` to format strings.  We can use `fstrings` or `urllib.urljoin`.